e2e/test/scenarios/dashboard-cards/click-behavior.cy.spec.js

```diff
@@ -179,6 +179,13 @@ describe("scenarios > dashboard > dashboard cards > click behavior", () => {
       editDashboard();
 
       getDashboardCard().realHover().icon("click").should("not.exist");
+      // When the default menu is selected, it should've visual cue (metabase#34848)
+      cy.get("aside")
+        .findByText("Open the Metabase drill-through menu")
+        .parent()
+        .parent()
+        .should("have.attr", "aria-selected", "true")
+        .should("have.css", "background-color", "rgb(80, 158, 227)");
```

Comment:
This assertion fails and should not be here.
Notice that line 181 makes sure that it's not possible to open the `aside` for that dashboard card.

```suggestion
```

Reply:
Hi @kamilmielnik . I apologize, this assertion might be moved to the wrong place when I'm rebasing from the `master` branch earlier this year. I moved the assertion to the correct place.